iceoryx_posh/include/iceoryx_posh/internal/roudi/port_manager.hpp

```diff
@@ -140,11 +135,15 @@ class PortManager
     cxx::optional<RuntimeName_t>
     doesViolateCommunicationPolicy(const capro::ServiceDescription& service IOX_MAYBE_UNUSED) const noexcept;
 
+    void publishCurrentServiceRegistry() const noexcept;
```

Comment:
Please rename this to `publishServiceRegistry` a distinction between current registry and some other state would only make sense if we would distinguish different states we want to publish.